### ISSUE TITLE
Support for additional file types

### ISSUE THREAD

**Issue by scriptPilot** [owncloud/files_texteditor#182]
Hello, I have edited the editor.js file to add support for phpx (php) files and added phpx to js/vendor/ace/src-noconflict/ext-modelist.js as well. I added the mime type "text/php-x" and the assignment phpx => php. But after purging the browser cache and reload ownCloud, there is still no support for the phpx files and the file is downloaded. Where I have to adjust the code as well? Regards!

**Comment by RealRancor** [comment]
Maybe this hint still works: https://github.com/owncloud/files_texteditor/issues/109

**Comment by scriptPilot** [comment]
Thank you for that hint!

**Comment by Kycko** [comment]
Hey guys, it also would be a good idea to have an ability to edit text files **without** any extension. I mean such files as _.gitconfig_ or _.zshrc_, or any other file

**Comment by scriptPilot** [comment]
Maybe it would be an idea to make it configurable in the preferences, which kind of files are editable and which one are downloaded.

**Comment by Kycko** [comment]
Yup. As I know, while using Dropbox you can not edit such files, but you can watch it's contents in web interface - it's a good solution too

**Comment by scriptPilot** [comment]
Is there any approach to handle this request? Regards!

**Comment by DeepDiver1975** [comment]
# Contributions are welcome!

![this-is-open-source](https://cloud.githubusercontent.com/assets/1005065/18341601/faf97b6e-75aa-11e6-8b10-acbc28cc8f94.jpg)

**Comment by Kycko** [comment]
@DeepDiver1975 unfortunately I know only Python and Bash :(
